packages/web-app-files/src/components/AppBar/ViewOptions.vue

```diff
@@ -83,7 +83,7 @@ export default {
     },
 
     toggleIcon() {
```

Comment:
Please rename the computed prop `sidebarIconFillType` or something like that. IMO inlining the one line into the template would be fine as well.

Reply:
See suggested change on line 24